playground/worker/vite.config-iife.js

```diff
@@ -10,29 +10,12 @@ export default defineConfig({
   },
   worker: {
     format: 'iife',
-    plugins: [
-      workerPluginTestPlugin(),
-      {
-        name: 'config-test',
-        config() {
-          return {
-            worker: {
-              rollupOptions: {
-                output: {
-                  entryFileNames: 'assets/worker_entry-[name].js',
-                },
-              },
-            },
-          }
-        },
-      },
-    ],
+    plugins: () => [workerPluginTestPlugin()],
     rollupOptions: {
       output: {
         assetFileNames: 'assets/worker_asset-[name].[ext]',
         chunkFileNames: 'assets/worker_chunk-[name].js',
-        // should fix by config-test plugin
-        entryFileNames: 'assets/worker_-[name].js',
+        entryFileNames: 'assets/worker_entry-[name].js',
```

Comment:
Instead of directly setting this value, does it make sense to move the `config-test` plugin to the top-level plugin?

Reply:
Good idea! And it led me to find that we weren't testing this in the spec at all. See https://github.com/vitejs/vite/pull/14685/files#diff-05860c3e1e292c3c7890765f1f5b6e83b1d8de52414c23c2d289957f60feea99R80